src/main.cpp

```diff
@@ -429,7 +429,6 @@ bool isEasterMode(int day, int month, int year, int before_after_days)
         }
         return (month > easter_start_month || (month == easter_start_month && day >= easter_start_day)) &&
                (month < easter_end_month   || (month == easter_end_month   && day <= easter_end_day));
-        break;
     }
```

Comment:
Is it actually necessary to use a switch statement here? It seems like it might be cleaner to just use an if statement and that would remove the need for a break.